test/plugin/base_test.rb

```diff
@@ -40,6 +40,24 @@ def setup
   # generic attributes
   HOSTNAME = Socket.gethostname
 
+  WRITE_LOG_ENTRIES_URI = 'https://logging.googleapis.com/v2beta1/entries:write'
+
+  APPENGINE_SERVICE_NAME = 'appengine.googleapis.com'
```

Comment:
A class in a module - no. But we could pull those constants out into their own module and import them into both classes...